x-pack/plugins/serverless_search/public/application/components/connectors/elastic_managed_connector_coming_soon.tsx

```diff
@@ -0,0 +1,367 @@
+            justifyContent="center"
+            direction="column"
+            gutterSize="l"
+          >
+            <EuiFlexItem>
+              <EuiButtonEmpty
+                data-test-subj="serverlessSearchElasticManagedConnectorEmptyBackButton"
+                iconType="arrowLeft"
+                onClick={() => navigateToUrl(`./`)}
+              >
+                {BACK_LABEL}
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiIcon size="xxl" type={connectorsIcon} />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle>
+                <h2>
+                  {i18n.translate('xpack.serverlessSearch.elasticManagedConnectorEmpty.title', {
+                    defaultMessage: 'Elastic managed connectors',
+                  })}
+                </h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiBadge color="accent">Coming soon</EuiBadge>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText textAlign="center" color="subdued">
+                <p>
+                  {i18n.translate(
+                    'xpack.serverlessSearch.elasticManagedConnectorEmpty.description',
+                    {
+                      defaultMessage:
+                        "We're actively developing Elastic managed connectors, that won't require any self-managed infrastructure. You'll be able to handle all configuration in the UI. This will simplify syncing your data into a serverless Elasticsearch project. This new workflow will have two steps:",
+                    }
+                  )}
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexGroup
+              alignItems="stretch"
+              justifyContent="center"
+              direction="column"
+              gutterSize="s"
+            >
+              <EuiFlexItem>
+                <EuiPanel color="subdued">
+                  <EuiFlexItem grow={false}>
+                    <EuiStepsHorizontal
+                      css={css`
+                        pointer-events: none;
+                      `}
+                      steps={horizontalSteps}
+                      size="s"
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexGroup>
+                    <EuiFlexItem>
+                      <EuiFlexGroup
+                        justifyContent="flexStart"
+                        alignItems="center"
+                        direction="column"
+                      >
+                        <EuiFlexItem grow={false}>
+                          <EuiFlexGroup
+                            justifyContent="center"
+                            alignItems="center"
+                            direction="row"
+                            gutterSize="s"
+                          >
+                            <EuiFlexItem grow={false}>
```

Comment:
Instead of creating four different, identical sets use a `map` function to do this once:
```
randomConnectors.map(connector => (<EuiToolTip content={connector.name}>......
```

Reply:
The nuance here is that the rendered outcome should be something like: 
`connector icon 1` `connector icon 2 ` `Doc icon` `connector icon 3` `connector icon 4`